src/nvim/memline.c

```diff
@@ -619,18 +616,10 @@ void ml_timestamp(buf_T *buf)
   ml_upd_block0(buf, UB_FNAME);
 }
 
-/*
- * Return FAIL when the ID of "b0p" is wrong.
- */
-static int ml_check_b0_id(ZERO_BL *b0p)
+/// Checks whether the ID of "b0p" is correct.
```

Comment:
Now that I think of it, all the `ml_check_b0` functions check whether the swap file is in a known/correct format.. I changed the comment to at least be consistent with the `ml_check_b0_strings` function below.